RocketDataTests/CollectionDataProviderTests/SharedCollectionDataProviderTests.swift

```diff
@@ -790,6 +791,7 @@ class SharedCollectionDataProviderTests: SharedCollectionTests {
             finishCacheLoad = {
                 completion(initialModels, nil)
             }
```

Comment:
eh...doesn't work great because they I need to pull out the model creation logic which I think does belong in this block.

I want `let initialModels: [Any] = [ParentModel(id: 0)]` to be passed on 'automatically'.